rest/call/list-get-example-7/list-get-example-7.5.x.cs

```diff
@@ -0,0 +1,24 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var fromParameter = new PhoneNumber("client:tommy");
+        var calls = CallResource.Read(from: fromParameter);
```

Comment:
The same for `fromParameter`, the context is pulled from `from` as a named parameter.